[2024-10-02T19:00:32.148+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T19:00:32.197+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-02-29T00:00:00+00:00 [queued]>
[2024-10-02T19:00:32.514+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-02-29T00:00:00+00:00 [queued]>
[2024-10-02T19:00:32.515+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-02T19:00:32.535+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): carregar> on 2024-02-29 00:00:00+00:00
[2024-10-02T19:00:32.547+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=6631) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-02T19:00:32.549+0000] {standard_task_runner.py:63} INFO - Started process 6649 to run task
[2024-10-02T19:00:32.550+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_covid_europa_pipeline', 'carregar', 'scheduled__2024-02-29T00:00:00+00:00', '--job-id', '1414', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpl6sn9ies']
[2024-10-02T19:00:32.553+0000] {standard_task_runner.py:91} INFO - Job 1414: Subtask carregar
[2024-10-02T19:00:32.627+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-02-29T00:00:00+00:00 [running]> on host 99de7dfc3871
[2024-10-02T19:00:32.752+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_covid_europa_pipeline' AIRFLOW_CTX_TASK_ID='carregar' AIRFLOW_CTX_EXECUTION_DATE='2024-02-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-29T00:00:00+00:00'
[2024-10-02T19:00:32.757+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T19:00:32.837+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-02T19:00:32.838+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T19:00:32.853+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_covid_europa_pipeline, task_id=carregar, run_id=scheduled__2024-02-29T00:00:00+00:00, execution_date=20240229T000000, start_date=20241002T190032, end_date=20241002T190032
[2024-10-02T19:00:32.884+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-02T19:00:32.903+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-02T19:00:32.911+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-10-02T19:26:34.128+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T19:26:34.425+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-02-29T00:00:00+00:00 [queued]>
[2024-10-02T19:26:34.434+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-02-29T00:00:00+00:00 [queued]>
[2024-10-02T19:26:34.435+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-02T19:26:34.449+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): carregar> on 2024-02-29 00:00:00+00:00
[2024-10-02T19:26:34.459+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=10026) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-02T19:26:34.460+0000] {standard_task_runner.py:63} INFO - Started process 10042 to run task
[2024-10-02T19:26:34.465+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_covid_europa_pipeline', 'carregar', 'scheduled__2024-02-29T00:00:00+00:00', '--job-id', '2433', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline_agrupar_continente.py', '--cfg-path', '/tmp/tmp0b23q7pj']
[2024-10-02T19:26:34.470+0000] {standard_task_runner.py:91} INFO - Job 2433: Subtask carregar
[2024-10-02T19:26:34.523+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-02-29T00:00:00+00:00 [running]> on host 99de7dfc3871
[2024-10-02T19:26:34.622+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_covid_europa_pipeline' AIRFLOW_CTX_TASK_ID='carregar' AIRFLOW_CTX_EXECUTION_DATE='2024-02-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-29T00:00:00+00:00'
[2024-10-02T19:26:34.623+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T19:26:34.736+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-02T19:26:34.737+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T19:26:34.747+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_covid_europa_pipeline, task_id=carregar, run_id=scheduled__2024-02-29T00:00:00+00:00, execution_date=20240229T000000, start_date=20241002T192634, end_date=20241002T192634
[2024-10-02T19:26:34.794+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-02T19:26:34.810+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-02T19:26:34.814+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
